Steam/Interpreter.cs

```diff
@@ -718,7 +718,7 @@ public static class Interpreter
 
         static Interpreter()
         {
-            Culture = new CultureInfo("en-EN", false);
+            Culture = new CultureInfo(CultureInfo.CurrentCulture.LCID, false);
```

Comment:
Why not Invariant?

Reply:
it's of course a performance reason...

![image](https://user-images.githubusercontent.com/15030741/72877520-652a0f80-3cf9-11ea-8ab2-2bc122c9fa6c.png)

Reply:
normally I would say, as you've been notified, that it's a performance reason, but making some benchmarks to be absolutely sure, in try parse from strings, it doesn't make really any difference, it only makes a difference (and local culture is more performant) when using .ToLower, to upper etc.